go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/cosmos/go-bip39 v1.0.0
 	github.com/cosmos/ibc-go/v5 v5.0.1
 	github.com/davecgh/go-spew v1.1.1
-	github.com/ethereum/go-ethereum v1.10.19
+	github.com/ethereum/go-ethereum v1.10.25
```

Comment:
```suggestion
	github.com/ethereum/go-ethereum v1.10.26
```

Reply:
Lets do this in another PR cause this one has been stale for a long time. Will create a ticket for it